system/src/system_update.cpp

```diff
@@ -540,6 +540,7 @@ const char* module_function_string(module_function_t func) {
         case MODULE_FUNCTION_SYSTEM_PART: return "s";
         case MODULE_FUNCTION_USER_PART: return "u";
         case MODULE_FUNCTION_NCP_FIRMWARE: return "c";
+        case MODULE_FUNCTION_RADIO_STACK: return "a";
```

Comment:
A similar change will be needed in https://github.com/particle-iot/binary-version-reader

Reply:
https://github.com/particle-iot/binary-version-reader/pull/24
https://github.com/particle-iot/particle-cli/pull/493